test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/net/URITest.scala

```diff
@@ -337,10 +365,12 @@ class URITest {
   }
 
   @Test def should_provide_hashCode(): Unit = {
-    if (!executingInJVM) { // Fails on JDK6 and JDK7
-      assertEquals(new URI("http://example.com/asdf%6a").hashCode,
-          new URI("http://example.com/asdf%6A").hashCode)
-    }
+    assertEquals(new URI("http://example.com/asdf%6a").hashCode,
+      new URI("http://example.com/asdf%6A").hashCode)
```

Comment:
4 spaces of indentation (same for the two below)

Reply:
Fixed